contracts/starknet/src/tests/test_escrow_claim.cairo

```diff
@@ -0,0 +1,246 @@
+    }
+
+    fn deploy_erc20(
+        name: felt252, symbol: felt252, initial_supply: u256, recipent: ContractAddress
+    ) -> IERC20Dispatcher {
+        let erc20 = declare('ERC20');
+        let mut calldata = array![name, symbol];
+        Serde::serialize(@initial_supply, ref calldata);
+        calldata.append(recipent.into());
+        let address = erc20.deploy(@calldata).unwrap();
+        return IERC20Dispatcher { contract_address: address };
+    }
+
+    #[test]
+    fn test_happy_path() {
+        let (escrow, eth_token) = setup();
+
+        // check balance
+        assert(eth_token.balanceOf(escrow.contract_address) == 0, 'init: wrong balance');
+        assert(eth_token.balanceOf(MM_STARKNET()) == 0, 'init: wrong balance');
+
+        start_prank(CheatTarget::One(escrow.contract_address), USER());
+        let order = Order { recipient_address: 12345.try_into().unwrap(), amount: 500, fee: 0 };
+        let order_id = escrow.set_order(order);
+        stop_prank(CheatTarget::One(escrow.contract_address));
+
+        // check balance
+        assert(eth_token.balanceOf(escrow.contract_address) == 500, 'set_order: wrong balance ');
+        assert(eth_token.balanceOf(MM_STARKNET()) == 0, 'set_order: wrong balance');
+
+        // check Order
+        assert(order_id == 0, 'wrong order_id');
+        let order_save = escrow.get_order(order_id);
+        assert(order.recipient_address == order_save.recipient_address, 'wrong recipient_address');
+        assert(order.amount == order_save.amount, 'wrong amount');
+        assert(escrow.get_order_pending(order_id), 'wrong order used');
+
+        let mut l1_handler = L1HandlerTrait::new(
+            contract_address: escrow.contract_address,
+            function_name: 'claim_payment'
+        );
+
+        let mut payload_buffer: Array<felt252> = ArrayTrait::new();
+        Serde::serialize(@order_id, ref payload_buffer);
+        Serde::serialize(@order.recipient_address, ref payload_buffer);
+        Serde::serialize(@order.amount, ref payload_buffer);
+
+        l1_handler.from_address = ETH_TRANSFER_CONTRACT().into();
+        l1_handler.payload = payload_buffer.span();
+
+        l1_handler.execute().expect('Failed to execute l1_handler');
+
+        // check Order
+        assert(!escrow.get_order_pending(order_id), 'wrong order used');
+        // check balance
+        assert(eth_token.balanceOf(escrow.contract_address) == 0, 'withdraw: wrong balance');
+        assert(eth_token.balanceOf(MM_STARKNET()) == 500, 'withdraw: wrong balance');
+    }
+
+    #[test]
+    fn test_claim_batch_happy() {
+        let (escrow, eth_token) = setup();
+
+        // check balance
+        assert(eth_token.balanceOf(escrow.contract_address) == 0, 'init: wrong balance');
+        assert(eth_token.balanceOf(MM_STARKNET()) == 0, 'init: wrong balance');
+
+        let recipient_addresses = array![12345.try_into().unwrap(), 12346.try_into().unwrap(), 12347.try_into().unwrap()];
```

Comment:
These values should be configured mode tidily , creating the variable in constants.cairo and then using the variable name. This makes the test easier to follow.

For example, in constants.caio :
```
    fn ETH_USER_3() -> EthAddress {
        33.try_into().unwrap()
    }
```
And then using 
`let recipient_address = ETH_USER_3();`
( maybe it needs to be `ETH_USER_3().into()` )

Also don't forget you'll need to import this variable in the file that will use it

Reply:
Changed on afa63ea46e63cb1ba712a7337bba490331fda65d & 96136fc262e7fafdfa5bf3cf2d429bef9dcf4b3d